docs/gaia/installation.md

```diff
@@ -40,6 +40,37 @@ $ gaiad version
 $ gaiacli version
 ```
 
+### Install binary distribution via snap (Linux only)
+
+Gaia can be installed on various GNU/Linux distributions from the [Snapcraft.io](https://snapcraft.io/gaia) store:
+
+```bash
+$ sudo snap install gaia
+```
+
+Development builds are available through the `edge` channel:
+
+```bash
+$ sudo snap install --edge gaia
+```
+
+::: tip
+At the time of writing, only the following [architectures are supported](https://build.snapcraft.io/user/cosmos/cosmos-sdk): `amd64` `i386` `arm64` `armhf` `ppc64el` `s390x`.
```

Comment:
```suggestion
At the time of writing, all of the following [architectures are supported](https://build.snapcraft.io/user/cosmos/cosmos-sdk): `amd64` `i386` `arm64` `armhf` `ppc64el` `s390x`.
```

Reply:
what other architectures are there?

Reply:
If we built on Launchpad.net, we would likely support the following archs too: `armel` (ARMv7 soft-float), `hppa`, `ia64`, `lpia` (remember the Netbooks wave that hit the market a roughly a decade ago? :) ), `powerpc`, `sparc` (!!!), `sparc64`.

Reply:
haha OK